src/framework/components/light/component.js

```diff
@@ -87,6 +91,10 @@ pc.extend(pc, function () {
      * @property {Number} cookieIntensity Projection texture intensity (default is 1).
      * @property {Boolean} cookieFalloff Toggle normal spotlight falloff when projection texture is used. When set to false, spotlight will work like a pure texture projector (only fading with distance). Default is false.
      * @property {String} cookieChannel  Color channels of the projection texture to use. Can be "r", "g", "b", "a", "rgb" or any swizzled combination.
+     * @property {Number} cookieAngle Angle for spotlight cookie rotation.
+     * @property {pc.Vec2} cookieScale Spotlight cookie scale.
+     * @property {pc.Vec2} cookieOffset Spotlight cookie position offset.
+     * @property {Boolean} isStatic Mark light as non-movable (optimization)
```

Comment:
Hmmm... then we also need a good correction to this: https://github.com/playcanvas/engine/pull/697/files#diff-a01b339213b1ec54098b0d4161e1bceeR28